terraform/README.md

```diff
@@ -0,0 +1,35 @@
+# Deploy Online Boutique sample application on a GKE cluster
+
+This page walks you through the steps required to deploy the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application on a GKE ([Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine)) cluster using Terraform.
+
+## Pre-requisites
+
+Setting up the sample requires that you have a [Google Cloud Platform (GCP) project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#console), connected to your billing account.
+
+## Deploying the sample application
+
+Once you have ensured that all the pre-requisites are met, follow the steps below to create a GKE cluster and deploy the Online Boutique sample application:
+
+1. Clone the repo:
+`git clone https://github.com/GoogleCloudPlatform/microservices-demo.git`
+1. Move into the Terraform directory that has the installation scripts:
+`cd microservices-demo/terraform`
+1. Navigate into the *variables.tf* file and set the gcp_project_id variable to your Google Cloud project ID:
+`default = "<project_id_here>"`
+1. Initialize Terraform:
+`terraform init`
+1. See what resources will be created:
+    `terraform plan`
+1. Create the resources and deploy the sample:
+    `terraform apply`
+    1. Note: This may take a while, do not interrupt the process.
```

Comment:
```suggestion
    1. Note: This may take about 10 minutes. Do not interrupt the process.
```

Providing a more precise time estimate is a better user experience.
It took me about 11 minutes for `terraform apply` to finish.